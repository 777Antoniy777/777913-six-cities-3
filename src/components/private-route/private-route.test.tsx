import * as React from "react";
import * as renderer from "react-test-renderer";
import {BrowserRouter} from "react-router-dom";
import PrivateRoute from "./private-route";

// set mocha data
const path = `path`;
const render = () => {
  return (
    <div></div>
  );
};
const authorizationStatus = `AUTH`;
const condRedirect = `address`;
const linkRedirect = `redirect`;

it(`render PrivateRoute`, () => {
  const tree = renderer.create(
      <BrowserRouter>
        <PrivateRoute
          path={path}
          render={render}
          authorizationStatus={authorizationStatus}
          condRedirect={condRedirect}
          linkRedirect={linkRedirect}
        />
      </BrowserRouter>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
