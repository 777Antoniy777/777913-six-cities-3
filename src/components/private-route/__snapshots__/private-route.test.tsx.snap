// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render PrivateRoute 1`] = `null`;
